docker-entrypoint-initdb.d/200_or_setup_vacuum_full.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# This script sets up a weekly VACUUM FULL job to clean up database bloat
+# VACUUM FULL reclaims more space than regular AUTOVACUUM by rewriting the entire table
+
+# Check if VACUUM FULL is enabled (default is disabled)
+if [ "${OR_ENABLE_VACUUM_FULL:-false}" != "true" ]; then
+    echo "VACUUM FULL is disabled (OR_ENABLE_VACUUM_FULL is not set to 'true'). Skipping setup."
+    exit 0
+fi
+
+echo "Setting up VACUUM FULL job (OR_ENABLE_VACUUM_FULL=true)..."
+
+# Create scripts directory if it doesn't exist
+mkdir -p /var/lib/postgresql/scripts
+
+# Create the vacuum script that will run VACUUM FULL on all tables in the database
+cat > /var/lib/postgresql/scripts/vacuum_full.sh << 'EOF'
+#!/bin/sh
+
+# Log file for vacuum operations
+LOG_FILE="/var/lib/postgresql/vacuum_full.log"
+
+echo "$(date): Starting VACUUM FULL operation" >> "$LOG_FILE"
+
+# Get list of all tables in the database
+TABLES=$(psql -t -c "SELECT schemaname || '.' || tablename FROM pg_tables WHERE schemaname NOT IN ('pg_catalog', 'information_schema')" "${POSTGRES_DB:-postgres}")
+
+# Run VACUUM FULL on each table
+for TABLE in $TABLES; do
+    echo "$(date): Running VACUUM FULL on $TABLE" >> "$LOG_FILE"
+    psql -c "VACUUM FULL $TABLE" "${POSTGRES_DB:-postgres}"
+    echo "$(date): Completed VACUUM FULL on $TABLE" >> "$LOG_FILE"
+done
+
+echo "$(date): VACUUM FULL operation completed" >> "$LOG_FILE"
+EOF
```

Comment:
You need to escape `$` here but I would move this into a file and use the `ADD` command in `Dockerfile`, no issues around var escaping then either.

No reason to not always have this file it's the cronjob that should be configurable.